bpf-metrics-exporter/README.md

```diff
@@ -14,24 +14,32 @@ to correlate process IDs -> containers -> k8s pods.
 
 The following metrics are exported:
 
-- `bpf_program_size_jitted_bytes`: The size of the BPF program in bytes.
-- `bpf_program_size_translated_bytes`: The size of the BPF program in bytes.
-- `bpf_program_mem_bytes`: The amount of memory used by the BPF program in bytes.
-- `bpf_program_verified_instructions_total`: The number of instructions in the BPF program.
-
-
-The following will be added pending: https://github.com/open-telemetry/opentelemetry-rust/issues/1242
-
-- `bpf_programs_total`: The number of BPF programs loaded into the kernel.
-
-Labels:
-
-- `id`: The ID of the BPF program
-- `name`: The name of the BPF program
-- `type`: The type of the BPF program
-- `tag`: The tag of the BPF program
-- `gpl_compatible`: Whether the BPF program is GPL compatible
-- `load_time`: The time the BPF program was loaded
+- `bpf_programs`: Information on each loaded BPF Program
+    - Labels:
+        - `id`: The ID of the BPF program
+        - `name`: The name of the BPF program
+        - `type`: The type of the BPF program as a readable string
+        - `tag`: The tag of the BPF program
+        - `gpl_compatible`: Whether the BPF program is GPL compatible
+        - `load_time`: The time the BPF program was loaded
+- `bpf_maps`: Information of each loaded BPF Map
```

Comment:
As above, please rename to `bpf_map_{N}_info` and add a `program_id` so we can associate it with a program

Reply:
Also, where is `bpf_map_memory_bytes`? We should have that as a metric for sure.

Reply:
We don't have `bpf_map_memory_bytes` yet from aya, I'll need to go back and add that

Reply:
Done on the rename, still need to add bpf_map_memory_bytes to aya

https://github.com/aya-rs/aya/issues/850